app/classes/report/symbiota.rb

```diff
@@ -2,7 +2,14 @@
 
 module Report
   # Symbiota-style tsv report.
+  # https://symbiota.org/
+  # https://biokic.github.io/symbiota-docs/
+  # https://github.com/Symbiota/Symbiota
   class Symbiota < TSV
+    # These plus Column labels comprising
+    # Symbiota Standard Field names which are useful for our purposes
+    # https://biokic.github.io/symbiota-docs/editor/edit/fields/#standard-fields
+    # plus MO-specific fields that are useful for uploads to Symbiota portals
     def labels
```

Comment:
Is it possible to create a single object which defines, in a single line, (1) a column label and (2) the value of that column for a row, sort of like this fantasy:
```ruby
def labeled_column_value(row)
  {
    scientificName: row.name_text_name,
    scientificNameAuthorship: row.name_author,
    ...
    recordedBy: collector(row),
    etc.
  }
end
(`label` and `formatted_row` would be derived from the above.)

This would make it easer to write.

Reply:
@nathan: 
> Is it possible to create a single object which defines, in a single line, (1) a column label and (2) the value of that column for a row, sort of like this fantasy:
> 
> ```ruby
> def labeled_column_value(row)
>   {
>     scientificName: row.name_text_name,
>     scientificNameAuthorship: row.name_author,
>     ...
>     recordedBy: collector(row),
>     etc.
>   }
> end
> ```
> 
> (`label` and `format_row` would be derived from the above.) This would be easier to read and write than the existing long definitions of `label` and `format_row`.